php-practice_1.php

```diff
@@ -1,43 +1,143 @@
 <?php
 
+// Q1
+
+$name='畠山';
+echo '私の名前は「'.$name.'」です';
 
 // Q2 四則演算
+$num = 5 * 4;
 
+echo $num;
+echo "\n";
+echo $num / 2;
 
 // Q3 日付操作
+$today = date("Y年m月d日 H時i分s秒");
 
-
+echo '現在時刻は、' . $today . 'です。';
 
 // Q4 条件分岐-1 if文
+$device = 'w';
 
+if ($device === 'windows'|| $device === 'mac'){
+    echo'使用OSは、' . $device . 'です。';
+}else {
+  echo'どちらでもありません。';
+      }
```

Comment:
[nits]
インデントがずれています！
```suggestion
if ($device === 'windows'|| $device === 'mac'){
    echo'使用OSは、' . $device . 'です。';
} else {
  echo'どちらでもありません。';
}
```

Reply:
インデント見直しをして修正しました。